procs/sqlite3/generate_selected_entities.py

```diff
@@ -25,11 +25,12 @@
 }
 
 def handle_task(task, data_structure, task_proc_mapping):
-    try:
+    task_proc_mapping[task](data_structure)
+    """try:
```

Comment:
is there a reason this block is commented out? @bvenedik-scalefree

Reply:
We moved the equivalent process of this try except block to a different location. Was commented out during development. I suppose the commented block can be removed.